assignment4.html

```diff
@@ -0,0 +1,30 @@
+ <body>
+   <form name="form1" >
+  
+            <label> Name : </label> 
+        <input id="name" type="text"  required> <input id="email" type="email"  pattern=".+@globex.com" >      
+        <button type="submit" onclick="add()">Submit</button>
+        <button " type="button" onclick="show()">Show User</button>
+   
+    </form>
+	<script>
+	function add()
+	{
+		var obj = new Object();
+		var date = new Date();
+		obj.time = date.toLocaleString();
+		obj.name = document.getElementById("name").value;
+		obj.email = document.getElementById("email").value;
+		sessionStorage.setItem('obj', JSON.stringify(obj));

```

Comment:
This will store only single user data, We should use an array for storing users list.